src/main/java/com/hms/flexyosisoftconnector/configuration/OSIsoftTagNamingScheme.java

```diff
@@ -0,0 +1,179 @@
+package com.hms.flexyosisoftconnector.configuration;
+
+import com.hms_networks.americas.sc.extensions.logging.Logger;
+import com.hms_networks.americas.sc.extensions.string.StringUtils;
+import java.util.List;
+
+/**
+ * This class contains the tag naming scheme parsing for the tag name on OSIsoft to be set by the
+ * connector.
+ *
+ * <p>HMS Networks Inc. Solution Center

```

Comment:
Javadoc header should use

```
@ author HMS Networks, MU Americas Solution Center
```

(without the space between @ and author -- I didn't want to tag that person)